examples/gallery/images/grdclip.py

```diff
@@ -0,0 +1,35 @@
+"""
+Clip grid values
+----------------
+The :meth:`pygmt.grdclip` method allows to clip defined ranges of grid values.
+In the example shown below we set all elevation values (grid points) smaller
+than 0 m (in general the bathymetric part of the grid) to a common value of
+-2000 m via the ``below`` parameter.
+"""
+
+import pygmt
+
+fig = pygmt.Figure()
+
+# Load sample grid (1 arc minute global relief) and use area around the Hawaiian Islands
+grid = pygmt.datasets.load_earth_relief(resolution="01m", region=[-162, -153, 18, 23])
```

Comment:
The 1 arc minute global relief data is [NOT cached](https://github.com/GenericMappingTools/pygmt/blob/6a4b8715bf00cd42cae8d5405e35e08a4764e97c/pygmt/helpers/testing.py#L145) in the CI. We should either cache the 01m data or use a lower resolution earth relief.

Reply:
I would prefer to use a high resolution as the 1 arc minute global relief, otherwise the map looks bad. Will add the 01m data to cache in a separate PR.

Reply:
Or can we use a different location? I like the Hawaii example, but caching the 1 arc minute data (258MB) isn't very feasible... How about another island like Iceland (or another medium-ish sized one).

Reply:
> Or can we use a different location? I like the Hawaii example, but caching the 1 arc minute data (258MB) isn't very feasible... How about another island like Iceland (or another medium-ish sized one).

Sounds a better solution.

Reply:
I changed it to the 3 arc minute grid for the area around Iceland.

Reply:
The current cache size is 30.5MB (https://github.com/GenericMappingTools/pygmt/actions/runs/1064597430), so adding 27MB would make it 50.5MB. I'm still a bit hesitant to add it... How about adding just the tile that is downloaded. Running `gmt grdcut @earth_relief_03m -R-28/-10/62/68 -Gtemp.nc` gives me the tile name "N00W090.earth_relief_03m_p" in `.gmt/server/earth/earth_relief/earth_relief_03m_p` so maybe try just caching that?

Reply:
> 
> 
> The current cache size is 30.5MB (https://github.com/GenericMappingTools/pygmt/actions/runs/1064597430), so adding 27MB would make it 50.5MB. I'm still a bit hesitant to add it... How about adding just the tile that is downloaded. Running `gmt grdcut @earth_relief_03m -R-28/-10/62/68 -Gtemp.nc` gives me the tile name "N00W090.earth_relief_03m_p" in `.gmt/server/earth/earth_relief/earth_relief_03m_p` so maybe try just caching that?

Does adding "N00W090.earth_relief_03m_p" allow to automatically use only this tile for plotting?

Reply:
See another similar case:
https://github.com/GenericMappingTools/pygmt/blob/6a4b8715bf00cd42cae8d5405e35e08a4764e97c/pygmt/helpers/testing.py#L163

I think we could only cache such a sinngle tile.

Reply:
> 
> 
> See another similar case:
> 
> https://github.com/GenericMappingTools/pygmt/blob/6a4b8715bf00cd42cae8d5405e35e08a4764e97c/pygmt/helpers/testing.py#L163
> 
> I think we could only cache such a single tile.

I already recognized this one 😉 . My question is: Is that tile automatically used by ```pygmt.datasets.load_earth_relief()``` when the corresponding region is defined?

Reply:
> Is that tile automatically used by `pygmt.datasets.load_earth_relief()` when the corresponding region is defined?

Yes.